src/Collector/Twig/HttpMessageMarkupExtension.php

```diff
@@ -16,32 +16,24 @@
  */
 class HttpMessageMarkupExtension extends AbstractExtension
 {
-    /**
-     * @var ClonerInterface
-     */
-    private $cloner;
+    private ClonerInterface $cloner;
 
-    /**
-     * @var HtmlDumper
-     */
-    private $dumper;
+    private DataDumperInterface $dumper;
 
-    public function __construct(?ClonerInterface $cloner = null, ?DataDumperInterface $dumper = null)
+    public function __construct(ClonerInterface $cloner = null, DataDumperInterface $dumper = null)
```

Comment:
afaik php 8.3 requires nullable to always have `?`
```suggestion
    public function __construct(?ClonerInterface $cloner = null, ?DataDumperInterface $dumper = null),
```

Reply:
++

I changed php-cs-fixer config to accomplish that